tools/build_litecore.sh

```diff
@@ -104,7 +104,7 @@ case $LIB in
    # works on centos6 and several version of OSX
    mbedcrypto)
       cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DCMAKE_POSITION_INDEPENDENT_CODE=1 ../../$MBEDTLS_DIR
-      make -j $JOBS mbedx509 mbedcrypto mbedtls
+      make -j $JOBS mbedcrypto
```

Comment:
This is my bad.  We shouldn't change this, in Lithium.  It is not necessary and therefore, not minimal.  We *should* make the change in Helium, but not here.

Sorry about that.